Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1555,6 +1556,19 @@ PopoverState HTMLElement::popoverState() const
     return popoverData() ? popoverData()->popoverState() : PopoverState::None;
 }
 
+
+bool HTMLElement::FireInvokeEvent(Element* invoker, const AtomString& action)
```

Comment:
```suggestion
bool HTMLElement::dispatchInvokeEvent(Element* invoker, const AtomString& action)
```
To match webkit style.